storage/database/db_manager.go

```diff
@@ -140,7 +139,7 @@ type DBManager interface {
 	WritePreimages(number uint64, preimages map[common.Hash][]byte)
 
 	// below operations are used in main chain side, not service chain side.
-	WriteChildChainTxHash(ccBlockHash common.Hash, ccTxHash common.Hash)
+	WriteChildChainTxHash(scBlockHash common.Hash, scTxHash common.Hash)
```

Comment:
How about setting same name policy?

Reply:
@jimni1222 Good point. @KimKyungup @jeonghonah How do you think about @jimni1222 suggest? ServiceChain or ChildChain?

Reply:
ChildChain is right. :)